### PR TITLE
Add RF test case for MQTT health check c8y

### PR DESCRIPTION
**Pull request by gligorisaev**
Testing MQTT Health endpoints, both testcases added in one robot file

- MQTT_health_endpoints.robot

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
#1172 

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by PradeepKiruvale — APPROVED** [review]
LGTM

File: tests/RobotFramework/MQTT_health_check/MQTT_health_endpoints.robot [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+*** Variables ***
+${HOST}           192.168.100.110
+${USERNAME}       pi
+${PASSWORD}       thinedge
+${pid}
+*** Tasks ***
+
+Connect to Cumulocity
+    ${rc}=    Execute Command    sudo tedge connect c8y
```
Comment (PradeepKiruvale) [inline tests/RobotFramework/MQTT_health_check/MQTT_health_endpoints.robot:14]: So, the assumption is that the `thin-edge` is already is installed right? I feel it's difficult to run an individual test. Better to check if `thin-edge` exists on that box or not right?

**Review by rina23q — APPROVED** [review]
LGTM

**Comment by matthiasbeyer** [comment]
How is this run in CI?

**Pull request merged**